Dockerfile

```diff
@@ -0,0 +1,33 @@
+##### Run kartograafr in Docker.
+### TTD (Things To Do)
+# - a couple of settings should be overrideable by environment variables.  See TODOs below.
+# - This runs as root.  Cron may require further work if the root user is not allowed.
+
+FROM python:2.7
+
+RUN apt-get update && apt-get install -y cron
+
+######### set timezone
+# TODO: allow overriding the time zone setting with an environment variable.
+ENV TZ=America/New_York
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+#########
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /usr/local/apps/kartograafr
```

Comment:
WRT @dlhaines' [earlier comment](https://github.com/tl-its-umich-edu/kartograafr/pull/13/files#r148256530):

OK, I think I get it.  Let me check.  Correct me if I'm wrong:

* `WORKDIR x` specifies the default directory somewhat like `cd x` in Bash, but only affects arguments that are _in the container_.  Filenames as arguments to commands that follow are assumed to be relative to directory `x`, unless they have explicit absolute paths.

* `COPY` is always recursive.

* A Bash quasi-equivalent to the commands in your `Dockerfile` may be:

```bash
# 1. For the sake of example, simplistically assume the project's source
#    and the VM container are running on the same computer.
# 2. Imagine this script is running in the project's source directory.

# Specify destination directory, the VM container
WORKDIR='/usr/src/app'

# Copy requirements.txt from project source directory to VM container
cp -r requirements.txt ${WORKDIR}/./

# Run pip, using the copy of requirements.txt in the VM container
(cd ${WORKDIR}; pip install --no-cache-dir -r requirements.txt)

# Copy the entire project source directory to /usr/local/apps/kartograafr in VM container
cp -r . /usr/local/apps/kartograafr
```

Is that correct?  If so, couldn't this part of `Dockerfile` be shortened to:

```dockerfile
# Remove the "WORKDIR" directive.
# The remaining file arguments in this Dockerfile all have absolute paths.

COPY . /usr/local/apps/kartograafr
RUN pip install --no-cache-dir -r /usr/local/apps/kartograafr/requirements.txt
```

Thanks for helping me understand this better.

Reply:
As I look at my suggestion moments later, I see that I could add "`WORKDIR /usr/local/apps/kartograafr`", which would shorten the other two directives.  However, that would change the first directive to "`COPY . .`", which would look odd and be _difficult to grok_.

Reply:
I agree  that "COPY . ." while correct is confusing.  I like the explicit destination better and will use that in the future.